### ISSUE TITLE
Keyboard navigation and selection for WCAG

### ISSUE THREAD

**Issue by CharlesVanek** [standardnotes/forum#3706]
When navigating with keyboard only, pressing Enter on the Notes ## navigation should put cursor focus into the list of ## notes.  From there the user should be able to arrow up/down through the Notes and press Enter to be taken to the Note's Pane for editing.

**To Reproduce**
1. Open Stndard Notes on Windows; Tab to navigation "Notes ##" where ## is the number of notes you have
2. Press Enter on "Notes ##"

**Expected behavior**
Cursor positioning to move to the list of Notes (this does not occur) and the user should be able to navigate the list of Notes with up/down arrows.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser : N/A
 - Version: standard-notes-3.195.1-win-x64

**Additional context**
This is critical for WCAG compliance to navigate the Standard Notes application using screen readers for for people who can't use a pointing device and rely upon keyboard navigation.
In addition it's beneficial for productivity of all users.
